soccer_strategy/src/soccer_strategy/robot_controlled.py

```diff
@@ -93,6 +97,12 @@ def can_kick(self, ball: Ball, goal_position):
         nav_angle_diff = player_angle - robot_ball_angle
         distance_of_player_to_ball = np.linalg.norm(player_position - ball_position)
 
+        header = Header()
+        header.stamp = rospy.Time.now()
+        header.frame_id = f"robot{self.robot_id}/torso"
+        ang = Range(header, 1, abs(nav_angle_diff), 1, 2, 2)
```

Comment:
can't really understand the variables so maybe do 
Range(header=header, blabla=1, angle=abs(nav_angle_diff)) etc for better readibility

Reply:
or I prefer

r = Range()
r.header.stamp = rospy.Time.now()
r.header.frame_id = f"robot{self.robot_id}/torso"
r.blabla = 1

for better readability